### PR TITLE
CON-2318 training audit fix

### PR DESCRIPTION
**Pull request by MSilva95**
fix the audit for the ai branch exercise training

### CODE REVIEW

File: subjects/ai/training/audit/README.md [hunk 1]
```diff
@@ -16,29 +16,19 @@
 
 #### Exercise 1: MSE Scikit-learn
 
-##### The goal of this exercise is to learn to use `sklearn.metrics` to compute the mean squared error (MSE).
+###### Is the Mean Squared Error (MSE) calculated using `sklearn.metrics` library?
```
Comment (Frenchris) [inline subjects/ai/training/audit/README.md:19]: Article missing.
Is the Mean Squared Error (MSE) calculated using **THE** `sklearn.metrics` library?

File: subjects/ai/training/audit/README.md [hunk 2]
```diff
@@ -16,29 +16,19 @@
...
+###### Is the Mean Squared Error (MSE) correctly computed for the given `y_true` and `y_pred` values, and does the calculated MSE match the expected value?
 
 ---
 
 ---
 
 #### Exercise 2: Accuracy Scikit-learn
 
-##### The goal of this exercise is to learn to use `sklearn.metrics` to compute the accuracy.
+###### Is the accuracy computed using `sklearn.metrics` library?
```
Comment (Frenchris) [inline subjects/ai/training/audit/README.md:29]: Article missing.
Is the accuracy computed using **THE** `sklearn.metrics` library?

**Review by Frenchris — APPROVED** [review]
just two typos. otherwise LGTM.
